### PR TITLE
Utility function for merging two level set fields

### PR DESCRIPTION
**Pull request by mschreter**
This PR introduces the utility function to merge two level set fields based on boolean algebra (and solves Issue #108). The following cases are supported:

- intersection
- difference 
- union

image

### CODE REVIEW

File: include/meltpooldg/utilities/utilityfunctions.hpp [hunk 1]
```diff
@@ -5,15 +5,27 @@
...
 
 #include <deal.II/fe/fe_dgq.h>
+#include <deal.II/fe/fe_q.h>
 #include <deal.II/fe/fe_tools.h>
 
 #include <deal.II/lac/generic_linear_algebra.h>
 
+#include <deal.II/matrix_free/matrix_free.h>
+
+#include <meltpooldg/utilities/fe_integrator.hpp>
```
Comment (peterrum) [inline include/meltpooldg/utilities/utilityfunctions.hpp:15]: Not needed.

**Review by peterrum — APPROVED** [review]
Very nice!

**Comment by mschreter** [comment]
@peterrum: This PR is ready from my side. Maybe you could have a quick look at it? The necessary clean-up works will be part of a follow-up PR.

File: include/meltpooldg/utilities/utilityfunctions.hpp (hunk 1 above)
Comment (peterrum) [reply include/meltpooldg/utilities/utilityfunctions.hpp:15]: I don't see it where it is used. But nevertheless merge ;)

**Review by peterrum — APPROVED** [review]

**Pull request merged**